ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/external-lb/pre/cucushift-installer-rehearse-vsphere-ipi-external-lb-pre-workflow.yaml

```diff
@@ -0,0 +1,9 @@
+workflow:
+  as: cucushift-installer-rehearse-vsphere-ipi-external-lb-pre
+  steps:
+    pre:
+    - chain: cucushift-installer-rehearse-vsphere-ipi-external-lb-pre-provision
+    post:
+    - chain: cucushift-installer-rehearse-vsphere-ipi-deprovision
```

Comment:
The deprovision chain is not enough, or else will leave lb vm behind, need one more step `ipi-deprovision-vsphere-lb-external`

Reply:
pls prepare a dedicated deprovision chain for the pre external lb case.